packages/v4/src/content/design-guidelines/styles/typography/typography.md

```diff
@@ -22,132 +22,105 @@ We use Red Hat Display and Red Hat Text. Red Hat Display should be used for **la
-    </div>
-    <Spacer size="16" color="16" />
-    <div className="spacingItemStyle">
-      <TitleLevel styleProps={styleProps.body}>Body text should be RedHatText at 16px. It should have 24px leading.</TitleLevel>
-    </div>
-    <Spacer size="24" color="24" />
-    <div className="spacingItemStyle">
-      <div className="spacingTitleStyle">Second level title</div>
-    </div>
-    <Spacer size="8" color="8"/>
-    <div className="spacingItemStyle">
-      <TitleLevel styleProps={styleProps.second}>Design is a journey of discovery</TitleLevel>
-    </div>
-    <Spacer size="16" color="16" />
-    <div className="spacingItemStyle">
-      <TitleLevel styleProps={styleProps.body}>Body text should be RedHatText at 16px. It should have 24 px leading.</TitleLevel>
-    </div>
-    <Spacer size="24" color="24" />
-    <div className="spacingItemStyle">
-      <div className="spacingTitleStyle">Third level title</div>
-    </div>
-    <Spacer size="8" color="8"/>
-    <div className="spacingItemStyle">
-      <TitleLevel styleProps={styleProps.third}>Design is a journey of discovery</TitleLevel>
-    </div>
-    <Spacer size="16" color="16" />
-    <div className="spacingItemStyle">
-      <TitleLevel styleProps={styleProps.body}>1. Lists should use body text style with 8px between items.</TitleLevel>
-    </div>
-    <Spacer size="8" color="8" />
-    <div className="spacingItemStyle">
-      <TitleLevel styleProps={styleProps.body}>2. Lists should use body text style with 8px between items.</TitleLevel>
-    </div>
-    <Spacer size="8" color="8" />
-    <div className="spacingItemStyle">
-      <TitleLevel styleProps={styleProps.body}>&nbsp;&nbsp;&nbsp;&nbsp;a. Secondary items in the list should use 8px spacing.</TitleLevel>
-    </div>
-    <Spacer size="8" color="8" />
-    <div className="spacingItemStyle">
-      <TitleLevel styleProps={styleProps.body}>&nbsp;&nbsp;&nbsp;&nbsp;b. They use the same body text style as primary items.</TitleLevel>
-    </div>
-    <Spacer size="24" color="24" />
-    <div className="spacingItemStyle">
-      <div className="spacingTitleStyle">Fourth level title</div>
-    </div>
-    <Spacer size="8" color="8" />
-    <div className="spacingItemStyle">
-      <TitleLevel styleProps={styleProps.fourth}>Design is a journey of discovery</TitleLevel>
-    </div>
-    <Spacer size="16" color="16" />
-    <div className="spacingItemStyle">
-      <TitleLevel styleProps={styleProps.body}>Body text should be RedHatText at 16px. It should have a 24px leading.</TitleLevel>
-    </div>
-    <Spacer size="24" color="24" />
-    <div className="spacingItemStyle">
-      <TitleLevel styleProps={styleProps.small}>Small text style should be RedHatText at 14px.</TitleLevel>
-    </div>
-  </GridItem>
-</Grid>
+Note that use of the [text component](/components/text) will automatically apply the proper line height and spacing according to the specified text style.
```

Comment:
I'm not sure who this is geared towards, but fwiw the title component also applies the proper line height to text in the title based on the title/font size.